redact/tools/redact_file.py

```diff
@@ -63,8 +61,6 @@ def redact_file(file_path: str, out_type: OutputType, service: ServiceType, job_
                                               licence_plate_custom_stamp=licence_plate_custom_stamp,
                                               custom_labels=custom_labels)
         result = job.wait_until_finished().download_result()
-    except ConnectionError:
```

Comment:
should the new redactconect error be added here?

Reply:
No, because this `except` wasn't a real error handling. It only changed the error message. But I hope that the new error message (thrown in `redact_requests.py`) is descriptive enough.